src/extensions/replay/sessionrecording.ts

```diff
@@ -692,7 +696,10 @@ export class SessionRecording {
             if (!href) {
                 return
             }
+            this._lastHref = href
```

Comment:
You probably want to reset `_lastHref` if `href` is null or undefined? otherwise `_lastHref` might be with the wrong state I guess?